src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -6771,15 +6771,25 @@ End Class
             Dim intType As ITypeSymbol = comp.GetSpecialType(SpecialType.System_Int32)
 
             ' Illegal VB identifier and blank
-            Dim tuple2 = comp.CreateTupleTypeSymbol(ImmutableArray.Create(intType, intType), ImmutableArray.Create("123", ""))
-            Assert.Equal({"123", ""}, tuple2.TupleElementNames)
+            Dim tuple2 = comp.CreateTupleTypeSymbol(ImmutableArray.Create(intType, intType), ImmutableArray.Create("123", " "))
+            Assert.Equal({"123", " "}, GetTupleElementNames(tuple2))
```

Comment:
I didn't understand this change. The test was explicitly testing blank (not a space).